generator/findby.go

```diff
@@ -0,0 +1,192 @@
+package generator
+
+import (
+	"bytes"
+	"fmt"
+	"go/types"
+	"strings"
+)
+
+const (
+	tplFindByInclude = "\n\n// FindBy adds a new filter to the query that will require that\n" +
```

Comment:
we should use the ` for multiline strings instead of concatenated strings, it would be easier to read imho